scripts/listAIQuotas.ps1

```diff
@@ -0,0 +1,185 @@
+<#
+.SYNOPSIS
+    This script uses Azure CLI to list the remaining quotas for the candidate models in the candidate locations.
+.DESCRIPTION
+    This script lists the remaining quotas for the candidate models in the specified locations.
+    The script requires Azure CLI to be installed and logged in.
+.EXAMPLE
+    az login
+    az account set --subscription <subscription_id>
```

Comment:
how about move `az account set` into the script to save user config effort? for `az login` , not able to wrap it in script since interactively for account login right

Reply:
Yes. Users can input subscription by -s parameter and the script help set subscription if provided. Let me update the example to reduce confustion.